src/tests/ftest/osa/offline_drain.py

```diff
@@ -47,6 +50,10 @@ def run_offline_drain_test(self, num_pool, data=False, oclass=None, pool_fillup=
         if oclass is None:
             oclass = self.ior_cmd.dfs_oclass.value
 
+        # For testing multiple ranks as dmg parameters, use a list of ranks.
+        if multiple_ranks is True:
+            self.ranks = self.multiple_ranks
```

Comment:
Looks like `multiple_ranks` is a list in the config. Also, should use the arg passed in?
```suggestion
        # For testing multiple ranks as dmg parameters, use a list of ranks.
        if multiple_ranks:
            self.ranks = multiple_ranks
```

Reply:
Done.